request/builder.js

```diff
@@ -426,6 +436,7 @@ class ContractExecutionRequestBuilder {
     request.setContractArgument(this.contractArgument);
     request.setCertHolderId(this.certHolderId);
     request.setCertVersion(this.certVersion);
+    request.setFunctionArgument(this.functionArgument);
```

Comment:
Probably we can't do it. 
I checked Java `ContractExecutionRequest` stub generated by grpc and it throws NLP for null input as follows.

```
   public Builder setFunctionArgument(
        java.lang.String value) {
      if (value == null) {
    throw new NullPointerException();
  }
  
      functionArgument_ = value;
      onChanged();
      return this;
    }
```
Can you check the generated javascript stub just in case ?

Reply:
@feeblefakie 
I checked the generated stub, the flow is 

1. setFunctionArgument
```
/** @param {string} value */
proto.rpc.ContractExecutionRequest.prototype.setFunctionArgument = function(value) {
  jspb.Message.setProto3StringField(this, 5, value);
};
```

2. setProto3StringField
```
jspb.Message.setProto3StringField = function(msg, fieldNumber, value) {
  return jspb.Message.setFieldIgnoringDefault_(msg, fieldNumber, value, '');
};
```

3. setFieldIgnoringDefault_
```
jspb.Message.setFieldIgnoringDefault_ = function(
    msg, fieldNumber, value, defaultValue) {
  // TODO(b/35241823): replace this with a bounded generic when available
  goog.asserts.assertInstanceof(msg, jspb.Message);
  if (value !== defaultValue) {
    jspb.Message.setField(msg, fieldNumber, value);
  } else {
    msg.array[jspb.Message.getIndex_(msg, fieldNumber)] = null;
  }
  return msg;
};
```

4. setField
```
jspb.Message.setField = function(msg, fieldNumber, value) {
  // TODO(b/35241823): replace this with a bounded generic when available
  goog.asserts.assertInstanceof(msg, jspb.Message);
  if (fieldNumber < msg.pivot_) {
    msg.array[jspb.Message.getIndex_(msg, fieldNumber)] = value;
  } else {
    jspb.Message.maybeInitEmptyExtensionObject_(msg);
    msg.extensionObject_[fieldNumber] = value;
  }
  return msg;
};
```

It looks like it is possible to set null value to gRPC message field `function_argument`

I guess Java probably consider that setting `function_argument` to be null is unnecessary
 because if we don't set `function_argument` at all, it will be null by default.

Reply:
Oh, thank you for the investigation. (I expected that was way simpler.)
Ok, then the current code looks good !